src/EFCore.PG/Diagnostics/NpgsqlEventId.cs

```diff
@@ -57,107 +58,98 @@ private enum Id
-        ///     </para>
-        ///     <para>
-        ///         This event uses the <see cref="PropertyEventData" /> payload when used with a <see cref="DiagnosticSource" />.
-        ///     </para>
-        /// </summary>
-        //public static readonly EventId ByteIdentityColumnWarning = MakeValidationId(Id.ByteIdentityColumnWarning);
-
-        private static readonly string _scaffoldingPrefix = DbLoggerCategory.Scaffolding.Name + ".";
-        private static EventId MakeScaffoldingId(Id id) => new EventId((int)id, _scaffoldingPrefix + id);
-
-        /// <summary>
-        ///     A column was found.
-        ///     This event is in the <see cref="DbLoggerCategory.Scaffolding" /> category.
+// TODO: Is there any reason to keep this?
+//        private static readonly string _validationPrefix = DbLoggerCategory.Model.Validation.Name + ".";
+//        private static EventId MakeValidationId(Id id) => new EventId((int)id, _validationPrefix + id);
+//
+//        /// <summary>
+//        ///     <para>
+//        ///         No explicit type for a decimal column.
+//        ///     </para>
+//        ///     <para>
+//        ///         This event is in the <see cref="DbLoggerCategory.Model.Validation" /> category.
+//        ///     </para>
+//        ///     <para>
+//        ///         This event uses the <see cref="PropertyEventData" /> payload when used with a <see cref="DiagnosticSource" />.
+//        ///     </para>
+//        /// </summary>
+//        public static readonly EventId DecimalTypeDefaultWarning = MakeValidationId(Id.DecimalTypeDefaultWarning);
+//
+//        /// <summary>
+//        ///     <para>
+//        ///         A byte property is set up to use a SQL Server identity column.
+//        ///     </para>
+//        ///     <para>
+//        ///         This event is in the <see cref="DbLoggerCategory.Model.Validation" /> category.
+//        ///     </para>
+//        ///     <para>
+//        ///         This event uses the <see cref="PropertyEventData" /> payload when used with a <see cref="DiagnosticSource" />.
+//        ///     </para>
+//        /// </summary>
+//        public static readonly EventId ByteIdentityColumnWarning = MakeValidationId(Id.ByteIdentityColumnWarning);
+
+        static readonly string ScaffoldingPrefix = DbLoggerCategory.Scaffolding.Name + ".";
+        static EventId MakeScaffoldingId(Id id) => new EventId((int)id, ScaffoldingPrefix + id);
+
+        /// <summary>
+        /// A column was found. This event is in the <see cref="DbLoggerCategory.Scaffolding" /> category.
         /// </summary>
         public static readonly EventId ColumnFound = MakeScaffoldingId(Id.ColumnFound);
 
         /// <summary>
-        ///     The database is missing a schema.
-        ///     This event is in the <see cref="DbLoggerCategory.Scaffolding" /> category.
+        /// The database is missing a schema. This event is in the <see cref="DbLoggerCategory.Scaffolding" /> category.
```

Comment:
As long as you're fixing up xmldoc comments, why not align with EF Core and add the `<para>` tags?